lib/presentation/widgets/playlist_widget.dart

```diff
@@ -13,6 +16,11 @@ class PlayListWidget extends StatelessWidget {
       itemCount: playlist.length,
       itemBuilder: (_, i) => TrackWidget(
         trackModel: playlist[i],
+        onPressed: (bool isLiked) {
+          !isLiked
```

Comment:
switch it to isLiked to avoid unnecessary negation

Reply:
ok